Dockerfile

```diff
@@ -23,5 +23,4 @@ COPY README.md /home/sa_scanner
 WORKDIR /home/sa_scanner
 RUN pip install .
 
-WORKDIR /home/sa_scanner/
```

Comment:
Why is this line removed?

Reply:
cause it's obsolete

Reply:
Could you clarify this, the docs recommend using it: https://docs.docker.com/engine/reference/builder/#:~:text=Therefore%2C%20to%20avoid%20unintended%20operations%20in%20unknown%20directories%2C%20it%20is%20best%20practice%20to%20set%20your%20WORKDIR%20explicitly.